keycloak_add_users.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+import json
+import requests
+import sys
+import getopt
+
+class updateUserStruct:
+    def __init__(self, id, enabled, username, firstName, lastName, email, attributes):
```

Comment:
Consider using the `attrs` library. There are many examples of this in IQE and asking any of the senior devs should get you what you need.